project.clj

```diff
@@ -139,7 +139,9 @@
                                        :main-namespace "puppetlabs.puppetdb.core"
                                        :create-varlib true}
                                 :config-dir "ext/config/pe"}
-                  :version ~pe-pdb-version
+                  :plugins [[puppetlabs/lein-ezbake "0.2.8"
+                              :exclusions [org.clojure/clojure]]]
```

Comment:
We definitely should, but I'd suggest we do it in a later PR, so that we can move FOSS and PE to the 0.3 ezbake series independently, at least until the work to update the PE filesystem paths to match FOSS is complete.